content/Watsonx/WatsonxAI/files/rest-api.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Accesing Watsonx.ai via REST API\n",
+    "\n",
+    "In this lab, we will take a comprehensive look at making HTTP requests to access [Watsonx.ai's REST API](https://workbench.res.ibm.com/docs/api-reference) and learn how to use the functionality. This lab explore only a few of the many REST endpoints available so explore the REST API documentation to view the full list of capabilities.\n",
```

Comment:
I do not have access to the [Watsonx.ai's REST API](https://workbench.res.ibm.com/docs/api-reference) URL
This is the URL I've seen been used elsewhere: https://bam.res.ibm.com/docs/api-reference